src/Components/Dashboard/OpportunityPannel/RequirementField.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from "react";
+import { IoAdd } from "react-icons/io5";
+import { useSelector } from "react-redux";
+import { VscActivateBreakpoints } from "react-icons/vsc";
+import { RiDeleteBin6Fill } from "react-icons/ri";
+
+const RequirementField = ({
+	id,
+	label,
+	register,
+	errors,
+	setValue,
+	placeholder,
+}) => {
+	const [requirement, setRequirement] = useState("");
+	const [requirementList, setRequirementList] = useState([]);
+	const { opportunity, editOpportunity } = useSelector(
+		(state) => state.opportunity
+	);
+
+	const handleAddRequirement = () => {
+		if (requirement) {
+			setRequirementList([...requirementList, requirement]);
+			setRequirement("");
+		}
+	};
+	const handleRemoveRequirement = (index) => {
+		const updatedRequirementList = [...requirementList];
+		updatedRequirementList.splice(index, 1);
+		setRequirementList(updatedRequirementList);
+	};
+
+	useEffect(() => {
+		if (editOpportunity) {
+			setRequirementList(opportunity?.eligibilityCriteria);
+		}
+		register(id, { required: true, validate: (value) => value.length > 0 });
+		// eslint-disable-next-line
+	}, []);
+
+	useEffect(() => {
+		setValue(id, requirementList);
+		// eslint-disable-next-line
+	}, [requirementList]);
+
+	return (
+		<>
+			<div className="flex flex-col space-y-2">
+				<label className="label-style" htmlFor={id}>
+					{label} <sup className="text-pink-200">*</sup>
+				</label>
+				<div className="flex flex-col items-start space-y-2">
+					<input
+						type="text"
+						id={id}
+						value={requirement}
+						onChange={(e) => setRequirement(e.target.value)}
+						placeholder={placeholder}
+						className="input-style"
+					/>
+					<button
+						type="button"
+						onClick={handleAddRequirement}
+						className="font-semibold text-primary-600 flex flex-row items-center"
+					>
+						<span className="text-lg font-bold">
+							<IoAdd />
+						</span>
+						<span className="text-xs md:text-lg">
+							Include Eligibility Criteria
+						</span>
+					</button>
+				</div>
+
+				{requirementList.length > 0 && (
+					<ul className="mt-2">
+						{requirementList.map((requirement, index) => (
+							<li
+								key={index}
+								className="flex flex-col sm:flex-row items-center text-gray-500 md:text-justify mb-4"
+							>
+								<div className="flex flex-row justify-start self-start sm:self-auto items-center gap-x-1">
```

Comment:
Still, the alignment is not proper, also please change that circle SVG due its shape alignment is not looking clean, we can do something simple gray/black dot 
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/c44a57f6-5355-41bf-af81-7ed1eaececef)

We can fix this by making these changes 
- Remove the styling of `button` tag.
- We can change the colour of the SVG by editing it's `currentColor` attribute.

Here is the screenshot after applying changes that I suggested
![Screenshot from 2024-04-11 10-48-40](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/4503ab82-da16-4da8-9ca5-2199649a30cf)

Reply:
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/95533641/5c3d9367-c8e7-432e-ade8-fa854dd8a08f)
Look into it looking more accurate when the delete icon is at the end

Reply:
Same i am keeping the bullet point at start because when the content is more that should come at start else that will look at the center and description which will not look good .